solidity/dashboard/src/pages/TBTCApplicationPage.jsx

```diff
@@ -83,8 +83,8 @@ const TBTCApplicationPage = () => {
     unbondedValueWithdrawnCallback
   )
 
-  const onAuthorizationSuccessCallback = useCallback(
-    (contractName, operatorAddress) => {
+  const onSuccessCallback = useCallback(
+    (contractName, operatorAddress, isDeauthorization = false) => {
```

Comment:
what do you think about removing `isDeauthorization` and replacing with `isAuthorized = true`?  We have only 2 cases: `isAuthorized = true / false` it seems that it should cover `isDeauthorization` as well.